angular-rails-app/app/controllers/owners_controller.rb

```diff
@@ -3,10 +3,15 @@ class OwnersController < ApplicationController
    before_action :set_owner, only: [:show]
 
   def index
-    @owners = Owner.all
+    if params[:name]
+       @owners = Owner.where("name LIKE (?)", "%#{params[:name]}%").limit(6)
```

Comment:
yep, just saw the slack. I've created an [issue](https://github.com/chrishenry/deltanyc-heatseek/issues/104).